default.json

```diff
@@ -55,6 +55,25 @@
         "fcmProjectID": "wiregcmpushstaging",
         "isStaging": true
     },
+    "beta": {
+        "applicationId": "com.wire.internal",
+        "userId": "",
+        "_comment": "appName must match the value of 'app_name' in strings.xml",
+        "appName": "Wire Internal",
+        "launcherIcon": "ic_launcher_wire_internal",
```

Comment:
I think we'll need to add this resource too =D

Reply:
most of the values in this file are not used and simply copied from the old app, i will raise another PR with clen up